src/gitops.ts

```diff
@@ -9,27 +9,21 @@ import { printHeader, printLogs } from "./utils";
 import { applyPromiseToEntriesWithProgressBar } from "./progress";
 
 async function hasLocalChanges(dir: string) {
-	const res = await pcp.spawn("git", ["status", "--porcelain"], {
-		cwd: dir,
-		encoding: "utf8",
-	});
+	const res = await pcp.spawn("git", ["status", "--porcelain"], { cwd: dir, encoding: "utf8" });
```

Comment:
No catch for child process

Reply:
Not your change, but all git actions can fail afaik. Just resolve this if you don't wanna fix :)